react/Menu/index.jsx

```diff
@@ -124,10 +131,10 @@ class Menu extends Component {
         {...restProps}
       >
         {!component ? (
-          <MenuButton
+          <Button
             disabled={disabled}
             onClick={this.toggle}
-            label={text}
+            label={text || label}
             buttonClassName={buttonClassName}
```

Comment:
are we sure that we have in this `buttonClassName` the equal of : `className={cx('c-btn', styles['c-menu__btn'], buttonClassName)}` ? 🤔 

you say that there is no style but what about : 
```
.c-menu__btn
    font-size rem(16)
    border 0
    cursor pointer

.c-menu__btn:not([disabled]):not([aria-disabled=true]):hover
.c-menu__btn:not([disabled]):not([aria-disabled=true]):active
.c-menu__btn:not([disabled]):not([aria-disabled=true]):focus
    background-color transparent
```

Reply:
I removed the commit removing the MenuButton from this PR.